db/db_impl_open.cc

```diff
@@ -355,8 +355,6 @@ Status DBImpl::Recover(
     {
       unique_ptr<RandomAccessFile> idfile;
       EnvOptions customized_env(env_options_);
-      customized_env.use_direct_reads |=
-          immutable_db_options_.use_direct_io_for_flush_and_compaction;
```

Comment:
I believe we still want this check if use_direct_reads =false but use_direct_io_for_flush_and_compaction is true.

Reply:
yes you are right